apps/server/src/modules/board/controller/element.controller.ts

```diff
@@ -109,9 +110,10 @@ export class ElementController {
 	@Delete(':contentElementId')
 	async deleteElement(
 		@Param() urlParams: ContentElementUrlParams,
-		@CurrentUser() currentUser: ICurrentUser
+		@CurrentUser() currentUser: ICurrentUser,
+		@Headers('authorization') authorization: string
 	): Promise<void> {
-		await this.elementUc.deleteElement(currentUser.userId, urlParams.contentElementId);
+		await this.elementUc.deleteElement(currentUser.userId, urlParams.contentElementId, authorization);
```

Comment:
why do you need explicit access to the authorisation header?
The CurrentUser already contains the authenticated user, there should be no further need for the header.

Reply:
because later in the process we have to also call delete endpoint from tldraw.app (separate address) for deletion of drawing data that was received via websocket (this data is even in different db).
I didn't want to leave endpoint and whole controller without authorization.